changelog.d/21370-improvement-to-sysv-script.md

```diff
@@ -0,0 +1,4 @@
+fix to sysv script file `/distribution/init.d/vector`
+
+* Changed daemon call for a working to background
+* Added the new line pid file creation because that vector does not create or touched to pid file
```

Comment:
```suggestion
The SysV init script now starts Vector in the background.

authors: waltzbucks
```

I think the pidfile change is only required because of the addition of `nohup` no? Also adding the authors line to appease CI (and properly attribute 😄 ).

Reply:
I feel the same 😃